fastlane/lib/fastlane/action_collector.rb

```diff
@@ -15,7 +26,28 @@ def show_message
     end
 
     def determine_version(name)
-      super(name) || Fastlane::VERSION
+      self.class.determine_version(name)
+    end
+
+    # e.g.
+    #   :gym
+    #   :xcversion
+    #   "fastlane-plugin-my_plugin/xcversion"
+    def self.determine_version(name)
+      return super(name) if super(name)
```

Comment:
Let's assign the value of `super(name)` to a variable so that we don't make the method call 2x